modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view_content_dashboard_item_types.jsp

```diff
@@ -72,14 +71,11 @@ ContentDashboardItemTypeItemSelectorViewDisplayContext contentDashboardItemTypeI
 		var arr = [];
 
 		allSelectedElements.each(function () {
-			var row = this.ancestor('tr');
-
-			var data = row.getDOM().dataset;
+			var payload = JSON.parse(Liferay.Util.unescape(this.getDOM().value));
```

Comment:
I didn't see this in any part of the portal. I saw that in other cases using `row.setPrimaryKey`, there isn't this part of the code implementing `rowToggled`. Is this because those search containers has a row checker?

Reply:
We need to use a JSON as the primaryKey, because we don't have a unique primareKey (className + classPK) :(, in order to copy this value to the input fields value, we should escape it.

There are many use of unescape and JSON.parse in portal, what is the use you are not seen before?

Thanks!!!

Reply:
I understand the primaryKey and the escape/unescape thing, I mean that I didn't see this login in other `rowToggled` function. But it shouldn't be an impediment to use it 👍